docs/modules/ROOT/pages/references/parameters.adoc

```diff
@@ -2,6 +2,27 @@
 
 The parent key for all of the following parameters is `openshift4_console`.
 
+== `namespace`
+
+[horizontal]
+type:: string
+default:: `openshift-console`
+
+The namespace where the console is deployed.
```

Comment:
Might be worth adding a note that changing this parameter is not recommended and may break the component's functionality.